modules/react/select/stories/Select.stories.mdx

```diff
@@ -34,79 +34,145 @@ yarn add @workday/canvas-kit-react
 
 ### Basic Example
 
-Select should be used in tandem with [Form Field](/components/inputs/form-field/) to meet
-accessibility standards.
+`Select` supports a
+[dynamic API](/getting-started/for-developers/resources/collection-api/#dynamic-items) where you
+pass an array of items via the `items` prop and provide a render function to display the items. The
+items may be provided as an
+[array of strings](/getting-started/for-developers/resources/collection-api/#array-of-strings) or an
+[array of objects](/getting-started/for-developers/resources/collection-api/#array-of-objects).
+
+`Select` should be used in tandem with [Form Field](/components/inputs/form-field/) where the
+`inputId` prop matches an `id` on the `Select.Input` to meet accessibility standards. This ensures
+the `label` text from `FormField` is attached to the `Select.Input` and read out as a group for
+voiceover.
 
 <ExampleCodeBlock code={Basic} />
 
+Our example uses [React state](<(https://react.dev/learn/state-a-components-memory)>) to track the
+value of the `Select`.
+
+### Hoisted Model
+
+By default, `Select` will create and use its own model internally. Alternatively, you may configure
+your own model with `useSelectModel` and pass it to `Select` via the `model` prop. This pattern is
+referred to as
+[hoisting the model](/getting-started/for-developers/resources/compound-components/#configuring-a-model)
+and provides direct access to its `state` and `events` outside of the `Select` component.
```

Comment:
[Your original Select docs](https://5d854c26ba934e0020f5e98a-bdjnqbgofl.chromatic.com/?path=/docs/components-inputs-select--basic), for reference.

I updated the `Hoisted Model` section and modeled it after the [`Hoisted Model` section in Tabs](https://github.com/Workday/canvas-kit/blob/master/modules/react/tabs/stories/Tabs.stories.mdx#L59). I also updated the example itself to provide a use case for why you might want to hoist the model (example is contrived, but it should get the job done).